examples/40_advanced/example_resampling_strategy.py

```diff
@@ -43,11 +43,6 @@
     # Build and fit a classifier with default resampling strategy
     # ===========================================================
     api = TabularClassificationTask(
-        temporary_directory='./tmp/autoPyTorch_example_tmp_03',
-        output_directory='./tmp/autoPyTorch_example_out_03',
-        # To maintain logs of the run, set the next two as False
-        delete_tmp_folder_after_terminate=True,
-        delete_output_folder_after_terminate=True,
```

Comment:
actually there are more tasks defined in this example. Could you also remove the temporary and output directory there?